packages/server/api/routes/reviews.router.js

```diff
@@ -0,0 +1,62 @@
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controllers
+const reviewsController = require('../controllers/reviews.controller');
+
+/**
+ * @swagger
+ * /reviews:
+ *  get:
+ *    tags:
+ *    - reviews
+ *    summary: Get all reviews
+ *    description:
+ *      Will return all reviews.
+ *    produces: application/json
+ *    responses:
+ *      200:
+ *        description: Successful request
+ *      5XX:
+ *        description: Unexpected error.
+ */
+router.get('/', (req, res, next) => {
+  reviewsController
+    .getAllReviews()
+    .then((result) => res.json(result))
+    .catch(next);
+});
+
+/**
+ * @swagger
+ * /reviews/{productId}:
```

Comment:
I don't think this should be a path variable, I'd rather see it as a query parameter. 

Path segments are meant to express **resources**. So `/users` is the list of all user resources, `/users/{id}` gives you a specific User resource based on it's ID. 

So here, `/reviews` gives you a list of all reviews, and `/reviews/{id}` should give me the review with that ID but that's not what you mean here. 

You mean to **filter** the list of `reviews` to only those that have the matching product ID. So I would say `/reviews?product={ID}` instead.